docs/REST-API/18-PCL-Overview.md

```diff
@@ -0,0 +1,590 @@
+- Comparison of non-comparable types returns a `false` Boolean Evaluation With Warnings. See [Note About Warnings During Expression Evaluation].
+
+##### `in`
+
+Returns `true` when the first argument falls within the schedule specified by the second argument. Otherwise, returns `false`.
+
+```
+now in [schedule] -> [boolean]
+```
+
+The `in` operator is only for creating _scheduled_ conditions.  The first arg must be [`now`](#now). The second arg must be a `schedule`.
+
+On Jan 3, 2022 at noon in Los Angeles:
+```
+now in Mon,Wed,Fri 08:00:00 to 18:00:00 America/Los_Angeles -> true
+```
+
+##### `exists`
+
+Returns `true` if the path exists in the context, `false` otherwise.
+
+```
+[path] exists -> [boolean]
+```
+
+Generally, a `path` used in an expression will evaluate to the value of the path in the context. In this case, `exists` checks to see if the path exists rather than retrieving the value at the path.
+
+`exists` will still return `true` if the `path` exists even if the value at that path is set to `nil`.
+
+```json
+{
+  "a": {
+    "b": nil,
+    "c": 5
+  }
+}
+  
+a.b exists -> true
+a.b -> nil
+a.c exists -> true
+a.c -> 5
+a.d exists -> false
+a.d -> nil
+```
+
+##### `now`
+
+A built-in PCL function that has no arguments and returns a `datetime` representing the "current" time.  For consistency during the evaluation of a complex exression, a single UTC value is determined before the entire expression is evaluated and that value is returned by this function. Could be used for the `datetime` argument in the one-time or recurring schedule conditions in Event Orchestrations.
+
+```
+now > 2020-01-01 00:00:00 Etc/UTC
+now in Mon,Fri 09:00:00 to 17:00:00 America/New_York
+```
+
+## Note About Warnings During Expression Evaluation
+
+[Note About Warnings During Expression Evaluation]:#note-about-warnings-during-expression-evaluation
+
+Evaluation of a PCL sub-expression can result in evaluation warnings. Since PCL will be used as part of the Event processing pipeline, throwing an error and giving up on the entire event evaluation isn't helpful since there is no user available to handle the error.
+
+Instead, PCL accumulates and propogates the evaluation warnings and then evalulates each sub-condition as accurately as possible given the previous warnings. The final result of the PCL evaluation will be a boolean decision along with any warning messages gathered along the way.
+
+There are two basic types of evaluation warnings: non-boolean evaluations and boolean results with evaluation warnings. 
+
+### Non-Boolean Evaluation
+
+[Non-Boolean Evaluation]:#non-boolean-evaluation
+
+These are PCL sub-expressions where the results are ambiguous / non-sensical. A classic example is an invalid type comparison like `2 > 'two'`. 
+
+PCL treats non-boolean evaluation as a `false` [boolean response with a warning][Boolean Evaluation With Warnings].
```

Comment:
**Not a blocker:** If you're accepting my other changes, above, where we remove links to the `Note About Warnings...` section, then I think we should remove that section entirely, **only** keeping the `Non-Boolean Evaluation` section, which _is_ still relevant & useful for Event Orchestration customers.

```suggestion
### Non-Boolean Evaluation

[Non-Boolean Evaluation]:#non-boolean-evaluation

Evaluation of a PCL sub-expression can produce results are ambiguous / non-sensical. A classic example is an invalid type comparison like `2 > 'two'`.

Since PCL will be used as part of the Event processing pipeline, throwing an error and giving up on the entire event evaluation isn't helpful since there is no user available to handle the error.

So PCL treats all non-boolean evaluation as a `false`, allowing the PCL expression to be evaluated consistently as a whole, without error.
```